mlflow/pyfunc/__init__.py

```diff
@@ -431,7 +431,7 @@ def predict(*args):
     return pandas_udf(predict, result_type)
 
 
-def save_model(dst_path, loader_module=None, data_path=None, code_path=None, conda_env=None,
+def save_model(path, loader_module=None, data_path=None, code_path=None, conda_env=None,
```

Comment:
The method docs need to be updated to read

`:param path: The path to which to save the Python model.`